faqs/galaxy/sequencing_nanopore.md

```diff
@@ -13,7 +13,7 @@ Nanopore sequencing has several properties that make it well-suited for our purp
 2. Long-read sequencing gives the ability to **span repetitive genomic regions**
 3. Long-read sequencing makes it possible to **identify large structural variations**
 
-![How nanopore sequencing works]({% link topics/metagenomics/images/plasmid-metagenomics-nanopore/sequence_method.jpg %} "Using nanopore sequencing, a single molecule of DNA or RNA can be sequenced without the need for PCR amplification or chemical labeling of the sample. (Image from: <a href="https://nanoporetech.com/sites/default/files/s3/white-papers/WGS_Assembly_white_paper.pdf?submissionGuid=40a7546b-9e51-42e7-bde9-b5ddef3c3512">Nanopore sequencing: The advantages of long reads for genome assembly</a>)") <br><br>
+![How nanopore sequencing works]({% link topics/metagenomics/images/plasmid-metagenomics-nanopore/nanopore_sequencing.png %} "Using nanopore sequencing, a single molecule of DNA or RNA can be sequenced without the need for PCR amplification or chemical labeling of the sample. (Image from: <a href="https://togotv.dbcls.jp/togopic.2020.01">Nanopore sequencing: The advantages of long reads for genome assembly</a>)") <br><br>
```

Comment:
the new Image from link isn't working for me.

Reply:
https://togotv.dbcls.jp/togopic.2020.01.html

Reply:
I am solving these comments right now :)

Reply:
done